cartography/intel/azure/iam.py

```diff
@@ -113,30 +116,29 @@ def list_tenant_users(client: GraphRbacManagementClient, tenant_id: str) -> List
 
 
 def transform_users(users_list: List[Dict], tenant_id: str) -> List[Dict]:
+    """
+    Transform user objects from MS Graph API format
+    Docs: https://learn.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-1.0
+    """
     users: List[Dict] = []
 
-    # User properties - https://learn.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-1.0
     for user in users_list:
+        # Only include properties from default response
         usr = {
-            'id': f"tenants/{tenant_id}/users/{user.object_id}",
-            'consolelink': azure_console_link.get_console_link(id=user.object_id, iam_entity_type='user'),
-            'object_id': user.object_id,
-            'user_principal_name': user.user_principal_name,
-            'email': user.mail,
-            'name': user.display_name,
-            'given_name': user.given_name,
+            'id': f"tenants/{tenant_id}/users/{user.id}",
+            'consolelink': azure_console_link.get_console_link(id=user.id, iam_entity_type='user'),
+            'objectType': 'microsoft.graph.user',
```

Comment:
there are a few issues here. 
1. db field names should match with earlier names. 
2. all fields should use snake case and not mixed cases.